ChangeLog.d/fix_cmake_gen_files.txt

```diff
@@ -1,3 +1,4 @@
 Bugfix
    * Fix an issue with CMake builds in releases with GEN_FILES turned off,
-     whereby missing generated files could be turned into symlinks to themselves.
+     whereby missing generated files could be turned into symlinks to
```

Comment:
Similar to before. I prefer "omitted" to "missing" here. Please consider changing it.

Reply:
That's the same as https://github.com/Mbed-TLS/mbedtls/pull/6689/files/d9d992f6715731cff41293611437c5c823a19c90..16ebf97cbb2798c27233807cb36adff7af100644#diff-580aa7c89d3f4c4c63d50bcd197ec7ae9301632754bc8f92d22fdae5c3f36414